AlphaWallet/Settings/ViewControllers/LocalesViewController.swift

```diff
@@ -16,15 +16,16 @@ class LocalesViewController: UIViewController {
     init() {
         super.init(nibName: nil, bundle: nil)
 
-        view.backgroundColor = Colors.appBackground
+        view.backgroundColor = GroupedTable.Color.background
 
         roundedBackground.translatesAutoresizingMaskIntoConstraints = false
         view.addSubview(roundedBackground)
 
         tableView.translatesAutoresizingMaskIntoConstraints = false
         tableView.delegate = self
         tableView.separatorStyle = .singleLine
-        tableView.backgroundColor = Colors.appBackground
+        tableView.backgroundColor = GroupedTable.Color.background
+        tableView.tableFooterView = UIView()
```

Comment:
What does the footer do?

Reply:
@hboon  I added it to make these table view screens look the same like settings screen, it removes separator lines for empty cells

Reply:
Oh very nice! Can you add a new static function to `UIView` and put it somewhere and call it, for future documentation. Something like:

```
static func tableFooterToRemoveEmptyCellSeparators() -> UIView {
  return .init()
}
```

Reply:
@hboon Done

Reply:
The one in `AssetDefinitionsOverridesViewController.swift` too :)

Reply:
> AssetDefinitionsOverridesViewController

oooh sorry, i missed it.

Reply:
@hboon check this one, should be done.